packages/core/src/config.js

```diff
@@ -28,6 +28,10 @@ export const configSchema = {
       },
       scope: {
         type: 'string'
+      },
+      deviceScaleFactor: {
+        type: 'integer',
+        default: 1
```

Comment:
When we add defaults to the schema like this, the option and its default will be added to freshly generated percy config files. Without this, we'd have to change the few logic gates slightly, but it won't end up in default config files.

Your call whether you think it should be included in a fresh config file, but I thought I'd at least mention how schema defaults can affect them.

Reply:
Given that, I'll remove the default. I also never used it further down like I was planning on so it's not bad to remove